api/envoy/config/filter/http/local_rate_limit/v2/BUILD

```diff
@@ -0,0 +1,13 @@
+# DO NOT EDIT. This file is generated by tools/proto_sync.py.
+
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_package")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_package(
+    deps = [
+        "//envoy/api/v2/core:pkg",
```

Comment:
v2 is now frozen, so please add directly to v3, thanks.

Reply:
ah ok.